index.html

```diff
@@ -0,0 +1,637 @@
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+
+            </div>
+
+            <div class="featured1">
+              <img src="images/9.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+
+            </div>
+          </div>
+          <div class="col-lg-4 col-md-6 top d-flex flex-column">
+            <h4 class="heading">TOP STORY</h4>
+            <img src="images/federal.png" alt="">
+            <p><a href="#">Federal Reserve Slashes Interest Rates to Near 0% Amid Coronavirus Pandemic</a></p>
+            <p>The central bank also urged banks to supply credit to the economy through the elimination of reserve requirements.</p>
+            <div class="culture">
+              <div class="culture1">
+                <img src="images/a.png" alt="">
+                <p class="top-p">Will I Get a Refund If My Trip Is Canceled Because of the Coronavirus?</p>
+                <p class="top-p1">Some insurance companies have been adjusting their policies in view of the outbreak, but travelers should check with their airlines, hotels and travel agents before filing a claim with insurers.</p>
+              </div>
+
+              <div class=" culture2 ">
+                <img src="images/b.png" alt="">
+                <p class="top-p">The Grandest Olympic Stadiums of All Time</p>
+                <p class="top-p1">While the fate of the 2020 Tokyo Olympics hangs in the balance due to the coronavirus pandemic, here's a look at other cities that have succeeded in creating structures that have stood the test of time and continue
+                  to be as awe-inspiring and groundbreaking as the games themselves.</p>
+              </div>
+
+              <div class=" culture3 ">
+                <img src="images/c.png" alt="">
+                <p class="top-p">'Outbreak' Enters Netflix's Top 10: 6 Quarantine-Ready Movies to Stream</p>
+                <p class="top-p1">If you're not looking for an escapist movie to watch during the coronavirus pandemic, try one of these films streaming online.</p>
+              </div>
+            </div>
+
+            <div class="more-stories">
+              <h4>MORE STORIES</h4>
+              <hr>
+              <div class="d-flex flex-column more-main">
+                <img src="https://d.newsweek.com/en/full/1574105/iran-coronavirus-ayatollah-hassan-rouhani.jpg?w=95&h=95&f=d9c023fd55255214598caecadddc9707" alt="">
+                <a href="#">Iran Senior Cleric Dies of Coronavirus, President Says Infections Peak</a>
```

Comment:
- [ ] The titles of the articles should be bold and have a red underline hover.

- [ ] The divisor should be gray.